src/LaurentMPoly.jl

```diff
@@ -168,3 +168,31 @@ For information about the many ways to specify `varnames...` refer to [`polynomi
 specification in [`AbstractAlgebra.@varnames_interface`](@ref).
 """
 laurent_polynomial_ring(R::Ring, s::Vector{Symbol})
+
+
+###############################################################################
+#
+#   Some operations
+#
+###############################################################################
+
+function is_unit(f::T) where {T<:LaurentMPolyRingElem}
+  # **NOTE** f.mpoly is not normalized in any way
+  characteristic(parent(f)) == 1 && return true  # coeffs in zero ring
```

Comment:
```suggestion
  is_trivial(parent(f)) && return true  # coeffs in zero ring
```
... and similarly elsewhere